packages/expo-modules-core/ios/Swift/Logging/LogType.swift

```diff
@@ -15,6 +15,27 @@ public enum LogType: Int {
   case error = 6
   case fatal = 7
 
+  public var asString: String {
```

Comment:
I think that only works with classes but I'll have a look

Reply:
- `print(LogType.warn)`
- `print(String(describing: LogType.warn))`
- `print("\(LogType.warn)")`

They all print the name of the enum case (not value, value is stored in `.rawValue` property), so I'm not sure what do you need it for? 🤔

Reply:
It turns out that you do need this if you are adding the `@objc` annotation. However, since we don't need to do this, you are correct, just `LogType.warn` is fine.